### PR TITLE
[sc-7419] Support local NucliaDB as destination in Desktop app

### PR DESCRIPTION
**Pull request by operramon**

### CODE REVIEW

File: libs/common/src/lib/label/label-field/label-field.component.ts [hunk 1]
```diff
@@ -25,15 +25,24 @@ export class LabelFieldComponent {
...
 
-  labelSets$ = this._kind.pipe(
-    switchMap((kind) =>
-      kind === LabelSetKind.RESOURCES ? this.labelsService.resourceLabelSets : this.labelsService.paragraphLabelSets,
+  labelSets$ = merge(
+    this._labelSets,
+    this._kind.pipe(
+      switchMap((kind) =>
+        kind === LabelSetKind.RESOURCES ? this.labelsService.resourceLabelSets : this.labelsService.paragraphLabelSets,
+      ),
```
Comment (mpellerin42) [inline libs/common/src/lib/label/label-field/label-field.component.ts:45]: I don't understand well why you do a merge here… if you provide the labelSets as an input, do you still need to get the labelsets by kind from `labelsService`?

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "nuclia",
-  "version": "2.6.20",
```
Comment (ebrehault) [inline package.json:3]: maybe it deserves to be 2.7.0 as that's a new feature

**Review by ebrehault — APPROVED** [review]

File: libs/common/src/lib/label/label-field/label-field.component.ts (hunk 1 above)
Comment (operramon) [reply libs/common/src/lib/label/label-field/label-field.component.ts:45]: `kind` and `labelSets` inputs are exclusive, only one of the should be provided.
Would you replace it by something like this ?
```
  ngOnInit() {
    if (labelsets input is provided) {
      this.labelsets$ = this._labelSets...
    }
    else {
      this.labelSets$ = this._kind...
    }
  }
```

Comment (mpellerin42) [reply libs/common/src/lib/label/label-field/label-field.component.ts:45]: OK, I think the fact those inputs are exclusive should be documented in the code then (a simple comment above the inputs would work)

Comment (operramon) [reply libs/common/src/lib/label/label-field/label-field.component.ts]: Fixed

**Review by mpellerin42 — APPROVED** [review]

**Pull request merged**